target/linux/bcm27xx/image/Makefile

```diff
@@ -106,6 +106,7 @@ define Device/rpi-2
 	raspberrypi,400 raspberrypi,4-compute-module raspberrypi,4-model-b \
 	raspberrypi,model-zero-2
   DEVICE_PACKAGES := \
+	brcmfmac-sdio-firmware-4345-bt \
```

Comment:
You are adding the firmware to bcm2709 subtarget, which is nice since that way we can have bluetooth for the RPi 4 @ 32 bits too, but you haven't enabled `CONFIG_SERIAL_DEV_CTRL_TTYPORT` in bcm2709 subtarget config. Is this option needed there too?
https://github.com/openwrt/openwrt/blob/master/target/linux/bcm27xx/bcm2709/config-5.15#L406